### PR TITLE
Coin Selection: new styles

### PR DESCRIPTION
**Pull request by MiguelMedeiros**
## Description
Changing coin selection setup screen layout.

## Preview

https://user-images.githubusercontent.com/5798170/179277537-bbe5155f-7550-4b9c-af56-2af44b9f3889.mp4

### CODE REVIEW

File: src/screens/Settings/CoinSelectPreference/index.tsx [hunk 1]
```diff
@@ -1,102 +1,113 @@
...
+						type: 'button',
+						onPress: async (): Promise<any> =>
+							updateSettings({ coinSelectAuto: true }),
+						hide: false,
+					},
+				],
+			},
+		],
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+		[selectedAutoPilot, coinSelectPreference],
```
Comment (limpbrains) [inline src/screens/Settings/CoinSelectPreference/index.tsx]: you can remove `// eslint-disable-next-line` and unused dependency `coinSelectPreference`

File: src/screens/Settings/CoinSelectPreference/index.tsx [hunk 2]
```diff
@@ -1,102 +1,113 @@
...
+							updateSettings({
+								coinSelectAuto: true,
+								coinSelectPreference: 'small',
+							}),
+						hide: !selectedAutoPilot,
+					},
+				],
+			},
+		],
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (limpbrains) [inline src/screens/Settings/CoinSelectPreference/index.tsx]: same here, `// eslint-disable-next-line` can be removed

File: src/screens/Settings/CoinSelectPreference/index.tsx [hunk 3]
```diff
@@ -1,102 +1,113 @@
...
+							updateSettings({ coinSelectAuto: false }),
+						hide: false,
+					},
+					{
+						title: 'Autopilot',
+						value: selectedAutoPilot,
+						type: 'button',
+						onPress: async (): Promise<any> =>
+							updateSettings({ coinSelectAuto: true }),
+						hide: false,
```
Comment (limpbrains) [inline src/screens/Settings/CoinSelectPreference/index.tsx:38]: You don't need to create async function onPress to call regular function updateSettings

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: src/screens/Settings/CoinSelectPreference/index.tsx (hunk 1 above)
Comment (MiguelMedeiros) [reply src/screens/Settings/CoinSelectPreference/index.tsx]: Fixed #[5c5348c](https://github.com/synonymdev/bitkit/pull/188/commits/5c5348c27b7e4c2a4c209e30ad9d5db1dcab7487)

File: src/screens/Settings/CoinSelectPreference/index.tsx (hunk 2 above)
Comment (MiguelMedeiros) [reply src/screens/Settings/CoinSelectPreference/index.tsx]: Fixed #[5c5348c](https://github.com/synonymdev/bitkit/pull/188/commits/5c5348c27b7e4c2a4c209e30ad9d5db1dcab7487)

File: src/screens/Settings/CoinSelectPreference/index.tsx (hunk 3 above)
Comment (MiguelMedeiros) [reply src/screens/Settings/CoinSelectPreference/index.tsx:38]: Fixed here  #[731bdda](https://github.com/synonymdev/bitkit/pull/188/commits/731bddafab19e100552d874831c60f6d25e5b3ad)

**Review by limpbrains — APPROVED** [review]

**Comment by MiguelMedeiros** [comment]
Open a new PR: https://github.com/synonymdev/bitkit/pull/195

**Pull request closed without merging**